build-support/bin/_generate_all_lockfiles_helper.py

```diff
@@ -114,7 +114,6 @@ def main() -> None:
             f"--mypy-protobuf-interpreter-constraints={repr(PythonProtobufMypyPlugin.default_interpreter_constraints)}",
             f"--mypy-protobuf-lockfile={PythonProtobufMypyPlugin.default_lockfile_path}",
             # Lambdex.
-            "--backend-packages=+['pants.backend.awslambda.python']",
```

Comment:
I think you'll probably still want to keep this line. Even though you define the subsystem in `pants/backend/python` now, unless a rule in that backend consumes the subsystem, it won't be registered.

You could even activate `awslambda` and `google_cloud_function`. Not really necessary, but maybe makes the association more explicit.

Reply:
Added both backends back in.